src/core/interfaces.ts

```diff
@@ -3,13 +3,13 @@ module Plottable {
    * Access specific datum property.
    */
   export interface Accessor<T> {
-    (datum: any, index: number, dataset: Dataset, plotMetadata: Plots.PlotMetadata): T;
+    (datum: any, index: number, dataset: Dataset): T;
   }
 
   /**
    * Retrieves scaled datum property.
    */
-  export type _Projector = (datum: any, index: number, dataset: Dataset, plotMetadata: Plots.PlotMetadata) => any;
+  export type _Projector = (datum: any, index: number, dataset: Dataset) => any;
```

Comment:
Now this is where type aliasing is probably handy.

`Accessor`s currently have the understanding that the scale has not affected their values whereas `_Projector`s do have that understanding that the scale has possibly affected the end value.

That said, if you think that differentiation is not enough of a merit for separation, possibly they can just be folded into `Accessor`